src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.async.txt

```diff
@@ -2,65 +2,130 @@
     2|       |
     3|       |// require-rust-edition-2018
     4|       |
-    5|      1|async fn f() -> u8 { 1 }
-    6|       |
-    7|       |async fn foo() -> [bool; 10] { [false; 10] }
-    8|       |
-    9|       |pub async fn g(x: u8) {
-   10|       |    match x {
-   11|       |        y if f().await == y => (),
-   12|       |        _ => (),
-   13|       |    }
-   14|      1|}
-   15|       |
-   16|       |// #78366: check the reference to the binding is recorded even if the binding is not autorefed
-   17|       |
-   18|       |async fn h(x: usize) {
-   19|       |    match x {
-   20|       |        y if foo().await[y] => (),
-   21|       |        _ => (),
-   22|       |    }
-   23|      1|}
-   24|       |
-   25|      1|async fn i(x: u8) {
-   26|      1|    match x {
-   27|      1|        y if f().await == y + 1 => (),
+    5|      1|async fn c(x: u8) -> u8 {
+    6|      1|    if x == 8 {
+    7|      1|        1
+    8|       |    } else {
+    9|      0|        0
+   10|       |    }
+   11|      1|}
+   12|       |
+   13|      0|async fn d() -> u8 { 1 }
```

Comment:
I'm looking for a one-line function that gets called but whose future is never polled.. I'd expect to see a line count of 1 with a 0 counter inside the body.

Reply:
I've added `m()`:

```
   90|       |
   91|      1|async fn m(x: u8) -> u8 { x - 1 }
                                      ^0
   92|       |
   93|      1|fn main() {
   94|      1|    let _ = g(10);
   95|      1|    let _ = h(9);
   96|      1|    let mut future = Box::pin(i(8));
   97|      1|    j(7);
   98|      1|    l(6);
   99|      1|    let _ = m(5);
  100|      1|    executor::block_on(future.as_mut());
  101|      1|}
```

Reply:
Done